src/Main.js

```diff
@@ -1,19 +1,26 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
+// Modules using `default` export must be imported...
```

Comment:
I'll add an advanced example without using `PlanarView` or `GlobeView` so I'll need to update exports.
It might end up in a separate PR though.